### PR TITLE
php8 fixes

### PR DESCRIPTION
**Pull request by bradymiller**
php8 fixes

### CODE REVIEW

File: interface/new/new_comprehensive.php [hunk 1]
```diff
@@ -586,7 +586,7 @@ function end_group()
                                   <?php
                                     foreach ($insurancei as $iid => $iname) {
                                         echo "<option value='" . attr($iid) . "'";
-                                        if (strtolower($iid) == strtolower($result3["provider"])) {
+                                        if (strtolower($iid) == strtolower($result3["provider"] ?? '')) {
```
Comment (bradymiller) [inline interface/new/new_comprehensive.php:589]:
```
if (!empty($result3["provider"]) && (strtolower($iid) == strtolower($result3["provider"]))) {
```

File: interface/orders/list_reports.php [hunk 2]
```diff
@@ -589,7 +589,7 @@ function doWait(e){
                     echo "  <td colspan='2'>&nbsp;</td>";
                 }
 
-                // Generate order columns.
+                // Generate order columns.$info['match']
```
Comment (bradymiller) [inline interface/orders/list_reports.php:592]: remove above line

File: interface/patient_file/summary/demographics_full.php [hunk 3]
```diff
@@ -591,7 +591,7 @@ function end_group()
                         <?php
                         foreach ($insurancei as $iid => $iname) {
                             echo "<option value='" . attr($iid) . "'";
-                            if (strtolower($iid) == strtolower($result3["provider"])) {
+                            if (strtolower($iid) == strtolower($result3["provider"] ?? null)) {
```
Comment (bradymiller) [inline interface/patient_file/summary/demographics_full.php:594]:
```
if (!empty($result3["provider"]) && (strtolower($iid) == strtolower($result3["provider"]))) {
```

File: interface/patient_file/summary/demographics_full.php [hunk 4]
```diff
@@ -832,9 +832,9 @@ function end_group()
                 <td class='required'>:</td>
                 <td colspan='2'>
                     <select class='form-control' name='i<?php echo attr($i); ?>accept_assignment'>
-                     <option value="TRUE" <?php if (strtoupper($result3["accept_assignment"]) == "TRUE") {
+                     <option value="TRUE" <?php if (strtoupper($result3["accept_assignment"] ?? '') == "TRUE") {
```
Comment (bradymiller) [inline interface/patient_file/summary/demographics_full.php:835]:
```
<option value="TRUE" <?php if (!empty($result3["accept_assignment"]) && (strtoupper($result3["accept_assignment"]) == "TRUE")) {
```

File: interface/patient_file/summary/demographics_full.php [hunk 5]
```diff
@@ -832,9 +832,9 @@ function end_group()
...
                             echo "selected"; }?>><?php echo xlt('YES'); ?></option>
-                     <option value="FALSE" <?php if (strtoupper($result3["accept_assignment"]) == "FALSE") {
+                     <option value="FALSE" <?php if (strtoupper($result3["accept_assignment"] ?? '') == "FALSE") {
```
Comment (bradymiller) [inline interface/patient_file/summary/demographics_full.php:837]:
```
<option value="FALSE" <?php if (!empty($result3["accept_assignment"]) && (strtoupper($result3["accept_assignment"]) == "FALSE")) {
```

**Comment by stephenwaite** [comment]
sisyphusian :)  

sisyphus

**Comment by bradymiller** [comment]
bringing this in.
summary:
-lots of php 8.0 fixes
-couple php 8.1 fixes
-turned of lots of the automated testing in php 8.1 until things not under our control (third party packages) have been addressed

**Pull request merged**